bin/circleci-install-shellcheck

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
I remember reading about some issue when using the portable shebang with `set -e`